ubuntu-2004.json

```diff
@@ -10,14 +10,17 @@
       "iso_checksum": "caf3fd69c77c439f162e2ba6040e9c320c4ff0d69aad1340a514319a9264df9f",
       "iso_checksum_type": "sha256",
 
-      "ssh_username": "ubuntu",
-      "ssh_password": "ubuntu",
-      "ssh_handshake_attempts": "20",
+      "ssh_username": "{{user `ssh_user`}}",
+      "ssh_password": "{{user `ssh_pass`}}",
+      "ssh_handshake_attempts": "30",
+      "ssh_timeout": "10m",
```

Comment:
I was curious that we'd need to do this, but I notice the wording is different in the docs:

> `ssh_timeout` (duration string | ex: "1h5m2s") - The time to wait for SSH to become available. Packer uses this to determine when the machine has booted so this is usually quite long. Example value: 10m.

https://www.packer.io/docs/builders/vmware/iso/#ssh_timeout

Are you finding that there's a lot of failures without specifying this?